Doc/library/sqlite3.rst

```diff
@@ -1435,15 +1442,29 @@ Cursor objects
       and there is no open transaction,
       a transaction is implicitly opened before executing *sql*.
 
+      Use :meth:`executescript` to execute multiple SQL statements.
 
    .. method:: executemany(sql, parameters, /)
 
-      Execute :ref:`parameterized <sqlite3-placeholders>` SQL statement *sql*
-      against all parameter sequences or mappings found in the sequence
-      *parameters*.  It is also possible to use an
-      :term:`iterator` yielding parameters instead of a sequence.
+      For every item in *parameters*,
+      repeatedly execute the :ref:`parameterized <sqlite3-placeholders>`
+      SQL statement *sql*.
+
       Uses the same implicit transaction handling as :meth:`~Cursor.execute`.
 
+      :param unicode sql:
+         A single SQL :abbr:`DML (Data Manipulation Language)` statement.
+
+      :param parameters:
+         A :term:`sequence` or :term:`iterator` of parameters
+         to bind with the placeholders in *sql*.
+         See :ref:`sqlite3-placeholders`.
+      :type parameters: :term:`sequence` | :term:`iterator`
```

Comment:
In typeshed we just use `Iterable` for this parameter rather than `Sequence | Iterator` — you could consider applying that simplification here too:

https://github.com/python/typeshed/blob/39487a8bab933a8c29edef5cf399de5fa8bb94cd/stdlib/sqlite3/dbapi2.pyi#L331

Reply:
Makes sense.